static/app/views/settings/organizationTeams/allTeamsRow.tsx

```diff
@@ -282,24 +282,19 @@ const TeamLink = styled(Link)`
 export {AllTeamsRow};
 export default withApi(AllTeamsRow);
 
-const TeamPanelItem = styled(PanelItem)`
+export const GRID_TEMPLATE = `
```

Comment:
We export a similar component with the same name from `teamMembersRow.tsx` - should we name this something more specific in case people autocomplete the import from the wrong file or just get confused?

Reply:
IMO this is fine. It's a very grid template specific to a page. I can't imagine people importing it randomly.